database/migrations/2021_04_06_051858_create_words_table.php

```diff
@@ -15,6 +15,7 @@ public function up()
     {
         Schema::create('words', function (Blueprint $table) {
             $table->id();
+            $table->string('word');
             $table->timestamps();
         });
```

Comment:
Next time create a new migration since you are modifying an already merged migration file.

If you change an already existing migration file then if ever i run `php artisan migrate`, the change will not be detected. I have to run `php artisan migrate:fresh` again